hugo/layouts/papers/list-entry.html

```diff
@@ -28,7 +25,7 @@
     {{ if eq hugo.Environment "development" }}
       <span class="badge badge-light align-middle">{{ .Params.anthology_id }}</span>
     {{ end }}
-    <strong><a class="align-middle" href="{{ .RelPermalink }}">{{ $paper.title_html | safeHTML }}</a></strong>
+    <strong><a class="align-middle" href="{{ $paper.url }}/">{{ $paper.title_html | safeHTML }}</a></strong>
     <br />
```

Comment:
Link to the canonical page (`https://aclweb.org/anthology/P19-1002/`) instead of the nested version. This will get rid of the double search results.

Reply:
Same concern as above: URL can refer to external content.

Reply:
Fixed. `url` now points to the page's Anthology URL; `pdf` points to the PDF (which can be internal or external).

Reply:
The downside of how this is now handled is that it completely breaks local browsing of the site, since links always point to the live anthology.

I see two options here:

1. Making the behaviour dependent on Hugo's "environment" variable, using `.RelPermalink` when in development mode and `$paper.url` otherwise. I don't really like this, as diverging behaviour has the potential to hide bugs.

2. Generating the flat structure directly with Hugo, as discussed in #480, instead of relying on redirects.